functests/dpdk/dpdk.go

```diff
@@ -355,6 +357,19 @@ func findOrOverrideSriovNetwork() {
 		CreateSriovPolicy(sriovDevice, sriovInfos.Nodes[0], 5, "dpdknic")
 		CreateSriovNetwork(sriovDevice, "dpdk-network", "dpdknic")
 	}
+
+	// When the dpdk-testing namespace is created it takes time for the network attachment definition to be created
+	// there by the sriov network operator
+	Eventually(func() bool {
+		netattachdef := &sriovk8sv1.NetworkAttachmentDefinition{}
+		err := client.Client.Get(context.TODO(), goclient.ObjectKey{Name: "dpdk-network", Namespace: namespaces.DpdkTest}, netattachdef)
+		if err != nil {
+			return false
+		}
+
```

Comment:
nit: can you remove the space?

Reply:
Also, you can directly return err and assert it not to be nil (I won't hold on any of these comments)